src/admin.rs

```diff
@@ -1,4 +1,5 @@
 use crate::manifest_store::ManifestStore;
+use log::warn;
```

Comment:
I'm getting:
```
warning: unused import: `log::warn`
 --> src/admin.rs:2:5
  |
2 | use log::warn;
  |     ^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
```

Not sure why since it is used on line 113? I'll dig into that after lunch.

Reply:
It's because the warn! call is only included when the `aws` feature is enabled, and we run some of the checks with all the features disabled. You can either conditionally include the import on `#[cfg(feature = "aws")]` or add some `warn!` call elsewhere in the file.

Reply:
Thanks!